src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -101,6 +101,7 @@ public static class ScriptConstants
         public const string AntaresScaleOutHeaderName = "X-FUNCTION-SCALEOUT";
         public const string AntaresColdStartHeaderName = "X-MS-COLDSTART";
         public const string SiteTokenHeaderName = "x-ms-site-restricted-token";
+        public const string FunctionsTokenHeaderName = "x-functions-token";
```

Comment:
The naming chosen here matches the scheme we use for key based auth, where we use "x-functions-key".